docs/source/dataframe-best-practices.rst

```diff
@@ -31,6 +32,19 @@ want to switch to Pandas at this point.
    result = result.compute()                # Convert to Pandas dataframe
    result...                                # Continue working with Pandas
 
+
+
+By default groupby methods return an object with only 1 partition. This is to 
```

Comment:
This doesn't quite feel like the right section for this. The header here is "Reduce, then use Pandas" so the assumption is that the user wants an in-memory object back.

I think a new section at the end of `dataframe-design.rst` is best.

Reply:
Done in https://github.com/dask/dask/pull/6318/commits/edfc7a50f719a04d0ca25e1579b71a2fd933cfd7